recipes/cdcs/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cdcs" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d0f5a7de7f7da6b2928b3e30962ea3fec3e8d77105742ea67336f8325590a415
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py2k]
```

Comment:
I think this recipe can be `noarch`. So please add `noarch: python` here and remove the `skip` line.